.travis.yml

```diff
@@ -26,7 +26,10 @@ services:
   docker
 language: python
 python:
-  - "3.5"
+  # The application runs against 3.5, which is set in the Dockerfile that is used to
```

Comment:
Ah, I see. This still isn't ideal though as it means we can't use the lint script from within the development environment.